subprojects/architecture-test/src/changes/accepted-public-api-changes.json

```diff
@@ -459,6 +459,363 @@
             "changes": [
                 "Method has been removed"
             ]
+        },
+        {
+            "type": "org.gradle.plugins.ide.idea.model.IdeaModule",
+            "member": "Method org.gradle.plugins.ide.idea.model.IdeaModule.getScalaPlatform()",
+            "acceptation": "Removal of deprecated method",
+            "changes": [
+                "Method has been removed"
+            ]
+        },
+        {
+            "type": "org.gradle.plugins.ide.idea.model.IdeaModule",
+            "member": "Method org.gradle.plugins.ide.idea.model.IdeaModule.setScalaPlatform(org.gradle.language.scala.ScalaPlatform)",
+            "acceptation": "Removal of deprecated method",
+            "changes": [
+                "Method has been removed"
+            ]
+        },
+
+
+        {
+            "type": "org.gradle.api.tasks.scala.IncrementalCompileOptions",
+            "member": "Class org.gradle.api.tasks.scala.IncrementalCompileOptions",
+            "acceptation": "Moved from 'gradle-language-scala.jar' to 'gradle-scala.jar' (this is not breaking and the checker should not report this)",
```

Comment:
It's unfortunate that we have to add all these entries here. The comparison currently works on a per-jar base. But the Gradle API is "one thing" at runtime and compile time (gradleApi()). So this is not a breaking change.

But fixing this in the comparison is not a simple change.

(The things that are actually removed are not in here, because "gradle-language-scala.jar" is no longer considered as we removed it completely)